template/_package.json

```diff
@@ -53,6 +60,9 @@
     <%_ } else { _%>
     "nuxt": "^2.0.0",
     <%_ } _%>
+    <%_ if (tsRuntime) { _%>
+    "@nuxt/typescript-runtime": "^0.2.10",
```

Comment:
/cc @clarkdo is it ok or better to use `^2.0.0` ?

Reply:
I think I may do it in another pr, it may contain loads of files change due to new lint rules

Reply:
https://github.com/nuxt/create-nuxt-app/pull/328/files/804aec1e6593f512498ff36db83c0d546da81630#diff-2c1b5c03cf7ce7b0571fd73c12c762e8R143

`"@nuxtjs/eslint-config-typescript": "^1.0.0"` uses v2 of `@nuxtjs/eslint-config` so should it be downgraded then ?

Reply:
https://github.com/nuxt/eslint-config/blob/master/packages/eslint-config-typescript/package.json#L15

May be worth bumping typescript one to same semantic version